### PR TITLE
[ISSUE-729] Fix issues in long e2e test suite

### LINKED ISSUE #729: CSI deletion in e2e tests is interrupted after error in CleanupLoopbackDevices
**Describe the bug**

When csi-baremetal-node pods are not created successfully and drivemgr container is not started, ExecShellInContainer function panics in cleanup stage. 

Example:
```
https://asd-ecs-jenkins.isus.emc.com/jenkins/job/csi-master-ci/567/testReport/(root)/CSI%20Suite/_sig_storage__CSI_Volumes__Driver__csi_baremetal___Testpattern__Dynamic_PV__filesystem_volmode___multiVolume__Slow__should_concurrently_access_the_single_volume_from_pods_on_different_node/

/usr/share/go/pkg/mod/k8s.io/kubernetes@v1.16.4/test/e2e/storage/testsuites/multivolume.go:306
failed to execute command in pod csi-baremetal-node-zjgqd, container drivemgr: unable to upgrade connection: container not found ("drivemgr")
Unexpected error:
    <*errors.errorString | 0xc000075fa0>: {
        s: "unable to upgrade connection: container not found (\"drivemgr\")",
    }
    unable to upgrade connection: container not found ("drivemgr")
occurred
/usr/share/go/pkg/mod/k8s.io/kubernetes@v1.16.4/test/e2e/framework/exec_util.go:101
```

**Environment (please complete the following information):**
 - Hardware: virtual
 - OS: Ubuntu 18
 - Version: 1.0.0

**Expected behavior**
Need to use ExecShellInPodWithFullOutput instead

### PR DESCRIPTION
**Pull request by safronovD**
### Resolves #729

- Replace `ExecShellInContainer` (raise panic if fail) with `ExecCommandInContainerWithFullOutput` (return error)
- Increase loopback device size to 2.1Gi in volume expansion (in e2e framework version 1.22 we need to setup 1Gi volumes in expansion tests)
- Reduce kind-nodes count (to decrease CI test VM's workload)

## PR checklist
- [x] Add link to the issue
- [x] Choose Project
- [x] Choose PR label
- [ ] New unit tests added
- [ ] Modified code has meaningful comments
- [ ] All TODOs are linked with the issues
- [ ] All comments are resolved

## Testing
n/a

### CODE REVIEW

**Review by mishoyama — APPROVED** [review]

**Review by misanthropicat — APPROVED** [review]

**Review by yurkov-anton — APPROVED** [review]

**Review by rkiselenko — CHANGES_REQUESTED** [review]
There is a config file for kind `https://github.com/dell/csi-baremetal/blob/master/test/kind/small-kind.yaml` can we use only one, they looks the same.

File: test/e2e/common/deploy.go [hunk 1]
```diff
@@ -241,7 +241,8 @@ func CleanupLoopbackDevices(ctx context.Context, f *framework.Framework) error {
 		return err
 	}
 	for _, pod := range pods {
-		f.ExecShellInContainer(pod, "drivemgr", "/bin/kill -SIGHUP 1")
+		stdout, stderr, err := f.ExecCommandInContainerWithFullOutput(pod, "drivemgr", "/bin/kill", "-SIGHUP", "1")
+		framework.Logf("Delete loopdevices in pod %s, stdout: %s, stderr: %s, err: %+v", pod, stdout, stderr, err)
```
Comment (rkiselenko) [inline test/e2e/common/deploy.go:245]: The format here should be 
```golang
framework.Logf("Delete loopdevices in pod %+v, stdout: %s, stderr: %s, err: %w", pod, stdout, stderr, err)
```

**Review by rkiselenko — CHANGES_REQUESTED** [review]

File: test/e2e/common/deploy.go (hunk 1 above)
Comment (safronovD) [reply test/e2e/common/deploy.go]: I missed it, thanks!

Comment (safronovD) [reply test/e2e/common/deploy.go]: Fixed

**Comment by safronovD** [comment]
> There is a config file for kind https://github.com/dell/csi-baremetal/blob/master/test/kind/small-kind.yaml can we use only one, they looks the same.

We need to have 3 nodes for [some scheduler tests](https://github.com/dell/csi-baremetal/blob/ef03ca5edbb359f475dd3c3b9157e53380b18db9/test/e2e/scenarios/scheduler-test.go#L194) (kind.yaml). But in Github Action using 2+ nodes is not stable due to not enough memory (small-kind.yaml)

**Review by rkiselenko — APPROVED** [review]

**Pull request merged**